lint.sh

```diff
@@ -77,6 +77,26 @@ check_bash(){
   done < <(find ${fargs[@]} -iname \*.sh)
 }
 
+check_naming_standards() {
+  # Limits lint to only gating unique files
+  # Excludes webhooktranslator as it has additional packaging
+  # and it's own tox
+  dirs_to_lint="pipeline_steps rpc_jobs scripts"
+  for _dir in ${dirs_to_lint}; do
+    for _file in $(find ${_dir} -type f -not -path '.*/.*'); do
+      python scripts/lint_naming_conventions.py \
```

Comment:
I don't really like the new python interpreter for every file approach. I'd prefer to either pass the list of dirs into python, or do the regex match in bash. http://wiki.bash-hackers.org/syntax/ccmd/conditional_expression#regular_expression_matching